daffodil-test-ibm1/src/test/scala/org/apache/daffodil/TresysTests.scala

```diff
@@ -158,9 +175,35 @@ class TresysTests {
 
   @Test def test_AZ000() { runnerAZ.runOneTest("AZ000") } // escape schemes
 
-  // Jira DFDL-1392
-  //  @Test def test_BA000() { runnerBA.runOneTest("BA000") } // escape schemes and delimiters
+  // Jira DFDL-1392 - Issue with escapeEscape character that is first and precedes an escape-block start.
+  // Is being removed, but should be preserved as it does not precede an escape character, nor an escape block end.
+  //@Test def test_BA000() { runnerBA.runOneTest("BA000") } // escape schemes and delimiters
+  //@Test def test_BB000() { runnerBB.runOneTest("BB000") } // occursCountKind stopValue
+
+  //DFDL-1010
+  //@Test def test_BE000() { runnerBE.runOneTest("BE000") } // unordered sequences
+  //@Test def test_BE001() { runnerBE.runOneTest("BE001") }
+
+  //DFDL-1010
+  //@Test def test_BF000() { runnerBF1.runOneTest("BF000") } // unordered sequences
+  //@Test def test_BF001() { runnerBF1.runOneTest("BF001") }
+
+  //@Test def test_BG000() { runnerBG.runOneTest("BG000") } // needs text numbers: advanced properties (DFDL-452)
+
+  //@Test def test_BC000() { runnerBC.runOneTest("BC000") } // text boolean type
```

Comment:
Some of these tests "should" work e.g., this one says in comment it needs text boolean type, which we have now.
There's no ticket associated with it (DFDL-XXX or DAFFODIL-XXX) so this will never get revisited..... Suggest we add a structured comment we can search for e.g., "// NO JIRA TICKET" so that we can easily take a pass and find these.

Reply:
I'm not sure how much that will really help that much since there's already alot of tests that are commented out but don't have associated bugs in the comment. Grepping for ``//.*@Test`` finds just over 300 instances of commented out bugs, but grepping for ``//.*(DFDL|DAFFODIL)-`` finds just under 200 tickets mentioned. Could be some are grouped together under a single issue, but I skimmed through commented out tests and quickly came across many that didn't have an associated bug.

Maybe we should instead create a new issue to go through commented out tests and either enable those that pass or create bugs for those that don't pass but don't have an associated bug?

Reply:
That's fine. I just wanted to highlight the issue since we no longer have an easy bulk way to just run all the debug tests. 

Also.... there might already be such a ticket.... vaguely recall.

Reply:
I couldn't find a ticket, so I've created [DAFFODIL-2095](https://issues.apache.org/jira/browse/DAFFODIL-2095) to test our commented out bugs and make sure those that fail have associated bug numbers.